include/openssl/http.h

```diff
@@ -33,6 +33,14 @@ extern "C" {
 # define OPENSSL_HTTP_PROXY "HTTP_PROXY"
 # define OPENSSL_HTTPS_PROXY "HTTPS_PROXY"
 
+/*
+ * We want to have this even in case of OPENSSL_NO_HTTP,
+ * it's a helper used in x509 name constraint (URI) checking.
+ */
```

Comment:
```suggestion
/* We want to have this even in case of OPENSSL_NO_HTTP */
```
As this is useful also in other non-HTTP contexts.